src/lib/serviceRoutinesV2/postSubscriptions.cpp

```diff
@@ -62,6 +62,10 @@ extern std::string postSubscriptions
     return answer;
   }
 
+  if (parseDataP->scr.res.attrsFormat == NGSI_NO_NOTIFICATION_FORMAT)
+  {
+    parseDataP->scr.res.attrsFormat = NGSI_V2_NORMALIZED; // FIXME PR: default notification format ?
```

Comment:
Here you have the modification of attrsFormat in a service routine.
Note that it is only changed if it not previously set, like a lifeline ...

And yes, the default here has to be NGSI_V2_NORMALIZED.
After all, we are in serviceRoutinesV2.

FIXME to be removed.